docs/ru/query_language/functions/geo.md

```diff
@@ -88,6 +88,37 @@ pointInPolygon((x, y), [(a, b), (c, d) ...], ...)
 Если точка находится на границе многоугольника, функция может возвращать как 0, так и 1.
 
 
+**Пример**
+
+``` sql
+SELECT pointInPolygon((3., 3.), [(6, 0), (8, 4), (5, 8), (0, 2)]) AS res
+```
+```
+┌─res─┐
+│   1 │
+└─────┘
+```
+
+## geoToH3
+
+Получает H3 индекс точки (lat, lon) с заданным разрешением
+
+```
+pointInPolygon(lat, lon, resolution)
+```
+
+**Входные значения**
+
+- `lat` - географическая широта. Тип данных — [Float64](../../data_types/float64.md).
+- `lon` - географическая долгота. Тип данных — [Float64](../../data_types/float64.md).
+- функция поддерживает также многоугольники с дырками (вырезанными кусками). Для этого случая, добавьте многоугольники, описывающие вырезанные куски, дополнительными аргументами функции. Функция не поддерживает неодносвязные многоугольники.
```

Comment:
Wrong text. BTW, you don't have to write russian docs if you don't know russian. English is enough.